test/dropdown_test.js

```diff
@@ -49,6 +49,8 @@ describe('DropdownController', () => {
       action.click()
       await nextFrame()
       expect(target.className.includes('hidden')).to.equal(false)
+      await aTimeout(10)
+      expect(target.className.includes('hidden')).to.equal(false)
```

Comment:
What's this testing?

Reply:
i adde this to show that the dropdown stays open. This test fails on the current main branch. Its because if there is two "action: click->dropdown#toggle" on separate element, it will open then immediately close. For some reason nextFrame() did not work, but aTimeout(10) does

Reply:
Gotcha, yeah. Would have caught that issue, but hopefully not something we need long-term.